.travis.yml

```diff
@@ -51,12 +51,18 @@ before_script:
   # only the arm64 fallback repo is unsigned and needs --allow-unauthenticated
   - sudo apt-get -y --allow-unauthenticated install librados-dev librbd-dev
 
+# Two stages for testing, each stage runs its jobs in parallel, but stages are
+# run after each other, unless the last stage fails.
+# Only run the deploy stage for the master branch on push (not pull_request)
+# events.
+stages:
+  - build testing
+  - e2e testing
+  - name: deploy
```

Comment:
do we need `-name: deploy` here or just `deploy` is fine here to maintain consistency with above stages

Reply:
even am not sure the stages will be retriggered when the PR is merged.

Reply:
> do we need `-name: deploy` here or just `deploy` is fine here to maintain consistency with above stages

I agree this looks weird, but this is the correct notation from what I understand from the docs.

Reply:
> even am not sure the stages will be retriggered when the PR is merged.

That happens when the merge is pushed to the master branch. At least, I expect that to be the case :crossed_fingers:

Reply:
then the changes look good. just we need to monitor the build logs once the PR is merged